From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Tue, 25 Jun 2013 12:23:52 +0200
Subject: Use format string

to fix

virt-viewer.c: In function 'virt_viewer_connect':
virt-viewer.c:686:13: error: format not a string literal and no format arguments [-Werror=format-security]
             g_warning(error->message);
---
 src/virt-viewer.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/virt-viewer.c b/src/virt-viewer.c
index 8d4ec62..f4d6aa5 100644
--- a/src/virt-viewer.c
+++ b/src/virt-viewer.c
@@ -683,7 +683,7 @@ virt_viewer_connect(VirtViewerApp *app)
 
     if (!virt_viewer_app_initial_connect(app, &error)) {
         if (error)
-            g_warning(error->message);
+            g_warning("%s", error->message);
         g_clear_error(&error);
         return -1;
     }
